integration_tests/package-lock.yml

```diff
@@ -0,0 +1,5 @@
+packages:
```

Comment:
should we gitignore package-lock.yml file?

Reply:
i think this file is actually intended to be version controlled, so that users of the same project have consistent installs -- could be wrong though!

Reply:
>this file is actually intended to be version controlled

correct in general, although I wonder whether it's valuable in integration test contexts 🤔 I think this means future minor/patch versions of utils won't be installed for testing, even when available, which I don't think is ideal.